### ISSUE TITLE
Vanished bookmarks

### ISSUE THREAD

**Issue by JohnFla** [rharel/webext-private-bookmarks#84]
For bug reports, please fill in:
 - Firefox version: 60.0.1
 - Operating system: Windows 7
 - Steps to reproduce: ?
 - What was the expected behavior: 
 - What was the actual behavior: I can't seem to find all my private bookmarks anymore. They are vanished, and the plugin asked me for a new password, as if it was the first time I set it up.. 

Can you help? Thank you

**Comment by rharel** [comment]
Strange. Did you update/reinstall Firefox recently? Is it possible another person went to PB's options page and used the "Clear Data" button? Anyway if you made backups (I'm referring to the export section in PB's options page), you can import like so:
1. Choose a password.
2. Unlock using the password you have just chosen.
3. In PB's options page, import a backup file using the password used when it was made.

If you did not make backups, your data is stored at a file `storage.js` in `C:\Users\<your username>\AppData\Roaming\Mozilla\Firefox\Profiles\<your profile>\browser-extension-data\private-bookmarks@rharel`, open that file in a text editor like notepad. Does it contain a `"back_folder":{...}` portion, where `...` includes among others a quite long string of letters and numbers? If it does, you could try what I suggested in https://github.com/rharel/webext-private-bookmarks/issues/82#issuecomment-410521933 If not, I'm afraid I can't be of more help :(

Let me know if there's anything else.

**Comment by JohnFla** [comment]
Hello Raoul,

Thank you for your prompt response.
I am not sure what happened, but I was able to recover my private bookmarks
by using the Firefox automatic backup.
Definitely I did not update Firefox recently.

Thank you for the workaround you suggested, I may need it in the future.

Cheers,


Giovanni Fiore

Ph.D. Applied Aerodynamics

<https://it.linkedin.com/in/giovanni-fiore-phd-9107344b>


Il giorno mer 29 ago 2018 alle ore 19:55 Raoul Harel <
notifications@github.com> ha scritto:
